### PR TITLE
IDE-3315 add support for custom http port

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalJBossBundle.java [hunk 1]
```diff
@@ -93,6 +93,11 @@ public String getHttpPort()
     }
 
     @Override
+    public void setHttpPort( String port )
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalJBossBundle.java]: Need to consult with Simon on how to set the http port for jboss.

**Comment by gamerson** [comment]
jenkins retest

File: tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalJBossBundle.java (hunk 1 above)
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/PortalJBossBundle.java]: I have told with Simon with it and he will work on jboss things after you merge this tomcat code, and now the server editor wouldn't show the port section if you select jboss server, i add this limit if we have no time to finish jboss stuff for 3.1 ga

**Comment by gamerson** [comment]
jenkins retest

**Pull request merged**